src/AbstractMappingTransformer.php

```diff
@@ -37,9 +38,10 @@ public function __construct($accessor = null)
      * @return array The output array
      * @throws OptimusException
      */
-    protected function transformFromMapping(array $mapping, array $data)
+    protected function transformFromMapping(array $mapping, array $data, $returnAllErrors = false)
```

Comment:
Please don't pass this option through the `transformFromMapping ` method itself. Use a ` parameter ` key inside the configuration file to add this option. It will be available in ` data['parameters'] `.

Reply:
Ok good idea !